modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentUtils.java

```diff
@@ -324,4 +332,36 @@ public static void send(Session ses, WebSocketResponse evt, long timeout, TimeUn
     public static List<String> trim(List<String> lst) {
         return F.isEmpty(lst) ? lst : lst.stream().map(String::trim).collect(toList());
     }
+
+    /**
+     * @param name Name.
+     * @param keyStorePath Key store path.
+     * @param keyStorePwd Key store password.
+     */
+    public static String getPasswordFromKeyStore(String name, String keyStorePath, String keyStorePwd) {
+        if (F.isEmpty(keyStorePath))
+            throw new IgniteException("The passwords key store path can't be empty!");
+
+        try {
+            KeyStore ks = KeyStore.getInstance("PKCS12");
```

Comment:
Why not JKS?

Reply:
We can't save secret keys in JKS, we can use JCEKS but keytool generate such warning:

The JCEKS keystore uses a proprietary format. It is recommended to migrate to PKCS12 which is an industry standard format using "keytool -importkeystore -srckeystore passwords.p12 -destkeystore passwords.p12 -deststoretype pkcs12".